komodo/symlink/sanity_check.py

```diff
@@ -82,10 +82,11 @@ def assert_root_nodes(link_dict):
     inferred_roots = _get_root_nodes(link_dict)
     if set(input_roots) != inferred_roots:
         raise AssertionError(
-            "The roots in the link-tree is not matching "
+            "The roots in the link-tree are not matching "
```

Comment:
I think, "The roots in the link-tree do not match " sounds more grammatical.

Reply:
Agreed!